diagnostic.js

```diff
@@ -41,7 +68,11 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+try {
+   db.houses.deleteOne( { "motto" : "As_High_as_Honor" } );
+} catch (e) {
+   print(e);
+}
```

Comment:
This is not correct. There is no `try` and `catch` in MongoDB scripts. Also, `deleteOne` deletes a document from the collection. That's what we are trying to accomplish.